deploy/release-validation-summary.py

```
@@ -0,0 +1,39 @@
+                'release-comparison', 'release-comparison-report')
+    passed = all(results.get(name, {}).get('result') == 'success' for name in required)
+    conclusion = 'success' if passed else 'failure'
+    lines = [f'# {tag} validation', '', f'**Validation: {conclusion.upper()}**', '',
+             f'[Release](https://github.com/{repository}/releases/tag/{tag}) · '
+             f'[Workflow](https://github.com/{repository}/actions/runs/{run})', '',
+             '| Job | Result |', '| --- | --- |']
+    lines += [f"| {name} | {results.get(name, {}).get('result', 'missing')} |" for name in required]
+    lines += ['', f'[Full e2e shard logs](e2e/{run}-{attempt}/)', '',
+              '[Latest stack flamegraphs](latest-stack.md)', '',
+              '[Latest three-way comparison](latest-comparison.md)', '',
+              f'[This attempt’s comparison data](comparisons/{run}-{attempt}/)', '',
```

Ruff (0.16.3)

[warning] String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)